.env.example

```diff
@@ -6,6 +6,7 @@ POSTGRES_DB=emails
 POSTGRES_USER=postgres
 POSTGRES_PORT=5432
 SERVER_IP=localhost
-RATE_SOURCE_API_KEY=
+CURRENCY_API_API_KEY=
```

Comment:
Maybe change name to CURRENCY_API_KEY

Reply:
I know it sounds weird, but 'CurrencyAPI' is a name of API itself https://currencyapi.com/